pkg/kubelet/cm/container_manager.go

```diff
@@ -31,7 +31,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/lifecycle"
 	"k8s.io/kubernetes/pkg/kubelet/pluginmanager/cache"
 	"k8s.io/kubernetes/pkg/kubelet/status"
-	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
framework is super generic. especially if used in the packages like kubelet (and it's about scheduler).

I would suggest doing "schedulerframework" instead.
[So effectively comparing to the original version: s/schedulernodeinfo/schedulerframework/]

Reply:
done for all occurrences outside pkg/scheduler